index.html

```diff
@@ -22,11 +22,55 @@ <h2>❗ Инструкция по выполнению теста</h2>
         конца, не прерывая его".
       </p>
     </div>
-    <div class="overlay hidden"></div>
+    <div class="overlay-instruction hidden"></div>
+
+    <div class="modal-form hidden">
+      <button class="close-modal-form">&times;</button>
+      <h2>📝 Введите свои данные</h2>
+      <form
+        action=""
+        class="userDataForm"
+        onkeydown="if(event.keyCode==13){return false;}"
```

Comment:
what is it for?

Reply:
it was for prevent form submit on ckick Enter button on kbrd. I think we don't need it already, because of this function
`  inputDataForm.addEventListener("submit", function (e) {
    e.preventDefault();`